File: hello-world/src/main/java/kz/sabyr/quarkus/service/impl/MyServiceImpl.java
package kz.sabyr.quarkus.service.impl;

import kz.sabyr.quarkus.service.MyService;

import javax.enterprise.context.ApplicationScoped;
import java.util.UUID;

@ApplicationScoped
public class MyServiceImpl implements MyService {
    @Override
    public String generateUUID() {
        return UUID.randomUUID().toString();
    }
}

File: hello-world-kotlin/src/main/kotlin/kz/sabyr/quarkus/helloworlkotlin/service/impl/MyServiceImpl.kt
package kz.sabyr.quarkus.helloworlkotlin.service.impl

import kz.sabyr.quarkus.helloworlkotlin.service.MyService
import java.util.*
import javax.enterprise.context.ApplicationScoped

@ApplicationScoped
class MyServiceImpl: MyService {
    override fun generateUUID() = UUID.randomUUID().toString()
}